src/components/NcRelatedResourcesPanel/NcRelatedResourcesPanel.vue

```diff
@@ -89,6 +89,20 @@ export default {
 			type: [String, Number],
 			default: null,
 		},
+		/**
+		 * Limits to specific resource type. i.e. any provider id implemented with `\OCA\RelatedResources\IRelatedResourceProvider::getProviderId()`
+		 */
+		resourceType: {
+			type: String,
+			default: null,
+		},
+		/**
+		 * Changes the number of results for the resourceType
```

Comment:
```suggestion
		 * Set the maximum number of resources to load
```
Limit does not only apply when resourceType is set